### ISSUE TITLE
Compile fails with NullReferenceException

### ISSUE THREAD

**Issue by kondotak** [ArxOne/MrAdvice#78]
Following the most recent change, the build (weave) will fail while the overall build will succeed.  There is a null reference exception with limited information posted in the output window.

![image](https://cloud.githubusercontent.com/assets/17506502/20088933/88dc24ec-a551-11e6-8ede-456a92fe8254.png)

**Comment by picrap** [comment]
So version 2.3.1 works, whereas 2.3.2 fails, is this right?

**Comment by picrap** [comment]
Version 2.3.3 changes the error message. Once you've installed it, can you paste it here?

**Comment by kondotak** [comment]
I've incorporated 2.3.4 into my main solution.  ILSpy confirms that the DLL's are weaved correctly with aspects incoroprated across the assembly.  dumpbin confirms that the pdb locations are set to absolute paths and the debugger is able to correctly load symbols from the pdb file.  So I think both of those issues are resolved, thanks for resolving so quickly.

I did see some warnings appear for some of the weaved assemblies (not all), something to the affect of "file not found", but I've been unable to reproduce the error for the last 30 minutes.  If I can re-produce I'll post.  But again, the dll's are woven correctly, and pdb is referenced with absolute.  Thanks again!
